src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  padding-top: 11px;
+  padding-bottom: 11px;
+  margin-right: 50px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  padding: 0;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  padding: 23px 0;
+}
+
+li:nth-child(n+2) {
```

Comment:
[Checklist ](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) 2nd item, don't use tagnames for styling, use classnames instead